client/my-sites/stats/stats-purchase/stats-purchase-checkout-redirect.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+	PRODUCT_JETPACK_STATS_MONTHLY,
+	PRODUCT_JETPACK_STATS_PWYW_YEARLY,
+	PRODUCT_JETPACK_STATS_FREE,
+} from '@automattic/calypso-products';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+const getStatsPurchaseURL = ( siteSlug: string, page: string ) => {
+	const purchasePath = `/checkout/${ siteSlug }/${ page }`;
+
+	return `https://wordpress.com${ purchasePath }`;
```

Comment:
Note: we'll need a `redirect_to` parameter to make sure it returns to the page we want after purchase. Refer to the Jetpack one for more information: https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/components/tools/get-product-checkout-url/index.ts#L10

Reply:
I believe this has been addressed? @grzegorz-cp

Reply:
Yes, it has been addressed, sorry for a late response.